src/constants/index.ts

```diff
@@ -11,6 +11,6 @@ export const KEY_CODES = {
 
 export const VIDEO_INPUT_QUALITY = {
   '360p': '360p (nHD) @ 15 fps (600 Kbps max)',
-  '540p': '540p (qHD) @ 15 fps (1.4 Mbps max)',
+  '540p': '540p (qHD) @ 15 fps (1.0 Mbps max)',
```

Comment:
Why do we change this?

Reply:
I thought this was a typo. Cause in JS SDK migration guide, the 540p's maximum bandwidth is 1.0 Mbps. I can revert this if it's not needed. Seems to be weird that 540p and 720p requires same maximum bandwidth.

Reply:
Oh I just chose 1000 as an example because the old `ChooseVideoinputQuality` has that parameter so i just chose a random value. 540p and 720p don't require the same max bandwidth but we don't need to change that behavior (as it is not in scope for this PR).

Reply:
Updated.